api/nodes/serializers.py

```diff
@@ -602,21 +604,33 @@ def create(self, validated_data):
         contributor.node_id = node._id
         return contributor
 
+
 class NodeContributorDetailSerializer(NodeContributorsSerializer):
     """
     Overrides node contributor serializer to add additional methods
     """
     id = ContributorIDField(required=True, source='_id')
+    index = ser.IntegerField(required=False, read_only=False)
+    # index = ser.IntegerField(required=False)
 
     def update(self, instance, validated_data):
+        index = None
+        if 'index' in validated_data:
+            index = validated_data.pop('index')
```

Comment:
Yeah, it's not strictly necessary here. Philosophically, it's because it's not a real value on the contributor, so if someone just passed validated_data as kwargs to update the contributor, it could break. As it stands, it's fine, since we aren't doing that.